demo/README.md

```diff
@@ -26,7 +26,7 @@ If you are working with source code, build the project and it will copy the requ
 ## Trust management
 The hashes for the default public keys present in iot-platform-sdk and supply-chain-tools are listed in following table.
 
-| Whitelisted Hash | SDO Component |
+| Allowlisted Hash | SDO Component |
```

Comment:
Unlike whitelist (which can be used a verb), allowlist can't be used as a verb.

'Hashes in allowlist'

Reply:
Done